### PR TITLE
SvaFinder: Be more correct with evangelical + catholic church in DE

### PR DESCRIPTION
**Pull request by baltpeter**
Mainly:

* Add two previously missing SVAs for the German evangelical church.
* Correct a few things about the catholic dioceses.
* Remove "Datenschutzbeauftragte des Verbandes der Diözesen Deutschlands". I misunderstood. That is an internal DPO, not an SVA.

Even with those changes, our handling of churches is still not correct. I'll open an issue for that. I've opened #989 to track that.

Thanks to Felix from artikel91.eu for making us aware of those mistakes cases!

### CODE REVIEW

File: src/Components/SvaFinder.tsx [hunk 1]
```diff
@@ -49,15 +49,43 @@ const steps: Record<string, Steps> = {
             // this allows us to determine which ones the BfDI is not responsible for.
             'bund-kk': 'debfdi',
             kirche: {
-                ev: 'deekdbfd',
+                ev: {
+                    // Source: https://dsbkd.de/geschaeftsverteilung/
+                    evdsbkd: 'deevpfalz',
+                    // Source: https://datenschutz.ekd.de/vernetzung/andere-evangelisch/
+                    evpfalz: 'dedsbkd',
```
Comment (zner0L) [inline src/Components/SvaFinder.tsx:56]: Is this correct or did you switch them around?

File: src/i18n/en.json [hunk 2]
```diff
@@ -834,14 +834,18 @@
         "sh": "Schleswig-Holstein",
         "thue": "Thuringia",
         "ev": "Evangelical church",
+        "ev-q": "Which church or Diakonisches Werk does your request concern?",
```
Comment (zner0L) [inline src/i18n/en.json:837]: I guess you didn't find a good translation here. What about just using "Diaconia" like [Wikipedia](https://en.wikipedia.org/wiki/Diaconia)?

**Review by zner0L — CHANGES_REQUESTED** [review]
Nice, thank you and the helpful user who helped us keep this correct. I still have two minor questions before we merge.

File: src/i18n/en.json (hunk 2 above)
Comment (baltpeter) [reply src/i18n/en.json:837]: I'm not well-versed with this, but I think "Diaconia" describes the theological concept rather than the institution.

**Review by baltpeter — COMMENTED** [review]
fix mixup

**Review by zner0L — APPROVED** [review]

**Pull request merged**